README.md

```diff
@@ -95,39 +95,41 @@ Confirm your PHP installation has curl enabled
 
 This quick example assumes that you have a local riak cluster running on port 8098
 
-    require_once('riak-php-client/src/Basho/Riak/Riak.php');
-    require_once('riak-php-client/src/Basho/Riak/Bucket.php');
-    require_once('riak-php-client/src/Basho/Riak/Exception.php');
-    require_once('riak-php-client/src/Basho/Riak/Link.php');
-    require_once('riak-php-client/src/Basho/Riak/MapReduce.php');
-    require_once('riak-php-client/src/Basho/Riak/Object.php');
-    require_once('riak-php-client/src/Basho/Riak/StringIO.php');
-    require_once('riak-php-client/src/Basho/Riak/Utils.php');
-    require_once('riak-php-client/src/Basho/Riak/Link/Phase.php');
-    require_once('riak-php-client/src/Basho/Riak/MapReduce/Phase.php');
-
-    # Connect to Riak
-    $client = new Basho\Riak\Riak('127.0.0.1', 8098);
-
-    # Choose a bucket name
-    $bucket = $client->bucket('test');
-
-    # Supply a key under which to store your data
-    $person = $bucket->newObject('riak_developer_1', array(
-        'name' => "John Smith",
-        'age' => 28,
-        'company' => "Facebook"
-    ));
-
-    # Save the object to Riak
-    $person->store();
-
-    # Fetch the object
-    $person = $bucket->get('riak_developer_1');
-
-    # Update the object
-    $person->data['company'] = "Google";
-    $person->store();
+```php
+require_once 'riak-php-client/src/Basho/Riak/Riak.php';
```

Comment:
What I'm trying to say is that if we want to suggest as part of the documentation that a developer needs to require all the files upfront like in the initially committed example, then we should advice the developer to use `require` rather than  `require_once`. 
The `*_once` php statements should be avoided as good practice.

That is what I was trying to address.

In addition I expressed my personal preference in the use of an autoloader whether this is a custom one or the composer one because that's what should be used and what people should learn to use.

Ultimately if we want to give flexibility and freedom to every developer we probably shouldn't even suggest what to use as part of the documentation but simply give indication about how to use the library itself and let the developer decide how to add it to the project.

@christophermancini what's your thought?